packages/babel-plugin-strip-runtime/src/index.ts

```diff
@@ -47,6 +51,44 @@ export default declare<PluginPass>((api) => {
               // If we used ESM it would blow up with CJS source, unfortunately.
             });
           }
+
+          if (this.opts.extractStylesToDirectory && this.styleRules.length > 0) {
+            // Build filename of the css file
+            const currentFilename = basename(filename);
+            const cssFilename = `${currentFilename.substring(
+              0,
+              currentFilename.lastIndexOf('.')
+            )}.compiled.css`;
+
+            if (!file.opts.generatorOpts?.sourceFileName) {
+              throw new Error(`Source filename was not defined`);
+            }
+            const sourceFileName = `${file.opts.generatorOpts.sourceFileName}`;
```

Comment:
is there a reason why `file.opts.generatorOpts.sourceFileName` is wrapped in a template string?

Reply:
good catch! That was something leftover from debugging I forgot to simplify

Reply:
- [ ] Remove template string